spec/index.bs

```diff
@@ -5475,7 +5500,14 @@ enum GPUStoreOp {
 
             **Returns:** {{undefined}}
 
-            Issue: Describe {{GPURenderEncoderBase/drawIndexed()}} algorithm steps.
+            Issue the following steps on the [=Device timeline=] of |this|.{{GPUObjectBase/[[device]]}}:
+            <div class=device-timeline>
+                If any of the following conditions are unsatisfied, make |this| [=invalid=] and stop.
+                <div class=validusage>
+                    - It is [$valid to draw$] with |this|.
+                    - |this|.{{GPURenderEncoderBase/[[index_buffer]]}} is not `null`.
```

Comment:
Indexed draw also needs to check that the indexFormat of the pipeline, if set, is equal to the indexFormat of the most recent setIndexBuffer. (bleh, extra draw time validation 🙁 )

Reply:
Done. Added a new internal slot to the pipeline (`[[strip_index_format]]`) to keep things from getting too wordy.